pyccl/tests/test_tk3d.py

```diff
@@ -165,12 +174,8 @@ def test_tk3d_spline_arrays_raises():
 
     # PR923 aims to change this bit of code; the assertion is there to remind
     # us to uncomment what is commented out.
-    assert not hasattr(tsp.__class__, "has_tsp")
```

Comment:
Do you want to remove the comment above?

Reply:
ah I forgot, thanks